code/modules/research/techweb/all_nodes.dm

```diff
@@ -1139,7 +1139,8 @@
 	boost_item_paths = list()
 	for(var/path in GLOB.uplink_items)
 		var/datum/uplink_item/UI = new path
-		if(!UI.item || !UI.illegal_tech || istype(UI, /obj/item/storage))
+		if(!UI.item || !UI.illegal_tech || ispath(UI.item, /obj/item/storage))
```

Comment:
Old

An empty duffel bag ![image](https://user-images.githubusercontent.com/3241376/112898326-4dd97580-90a6-11eb-93d9-70409e5be87e.png)

An empty box 
![image](https://user-images.githubusercontent.com/3241376/112898415-6b0e4400-90a6-11eb-929a-beb22a7038ad.png)

---
New 

![image](https://user-images.githubusercontent.com/3241376/112900177-ba557400-90a8-11eb-9410-e42bedb2a5aa.png)

Box items (not their contents) no longer give illegal tech

![image](https://user-images.githubusercontent.com/3241376/112900326-f4bf1100-90a8-11eb-8489-1b9076877941.png)